pymc3/tests/test_distributions.py

```diff
@@ -902,6 +902,7 @@ def test_normal(self):
             R,
             {"mu": R, "sigma": Rplus},
             lambda value, mu, sigma: sp.norm.logcdf(value, mu, sigma),
+            decimal=select_by_precision(float64=6, float32=2),
```

Comment:
`float32` precision of 2 digits sounds awfully bad. Are you sure about this?

Reply:
It's actually better than the logp test which uses a precision of 1! (Previous to this PR)

Reply:
This used to pass before because these tests were all seeded until very recently and the conditions with low precision were not appearing / failing since then